docs/guide/fragments.md

```diff
@@ -1,61 +1,75 @@
-# Fragments are the new macros
+# Template fragments
 
-[Nunjucks macros](https://mozilla.github.io/nunjucks/templating.html#macro) are great tools for sharing template code across files. They have one significant limitation, however, in that they run synchronously. If you include template tags in them that run asynchronously, those tags will seem to not run at all when it comes to rendering.
+We will often find that template code is reused across multiple files or that it simply gets overly complicated for a single file. **Template fragments** help solve this by splitting template code into smaller, managable pieces that can be used in other templates.
 
-This poses a challenge when using the new [async components](/guide/async-components) in Apostrophe 3. In addition, the [area syntax changed](/guide/major-changes.md#areas-and-pages) to support those async components, so areas won't work in synchronous macros either. To enable developers to continue writing maintainable, [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code, A3 has a new feature, **fragments**.
+::: note
```

Comment:
Trying to dwell less on their relationship to macros, but also get it out of the way for people who already known macros.